client/components/OrderHistory.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchOrders } from '../redux/orderHistory/orderHistory';
+
+class OrderHistory extends Component {
+  componentDidMount() {
+    this.props.fetchOrders();
+  }
+
+  render() {
+    const { orders } = this.props;
+    if (orders.length > 0) {
+      return (
+        <div>
+          <h1>Previous Orders:</h1>
+          <ul>
+            {orders.map((order) => (
+              <li key={order.id}>
+                Total: $
+                {order.total}
```

Comment:
can you make this .toFixed(2)? see Cart component for example (otherwise sometimes this becomes a ton of decimals)